pyproject.toml

```
@@ -19,6 +19,9 @@ classifiers = [
   "Programming Language :: Python",
 ]
 
+[tool.pytest.ini_options]
+pythonpath = ["."]
```

So specifically I was suggesting doing this:
```suggestion
testpaths = ["tests"]
```
Does this not do the right thing?

---

According to https://docs.pytest.org/en/stable/reference/customize.html, this should work but does not. At least here.

---

I think I have found a less magic solution for this. From the [pytest docs](https://docs.pytest.org/en/stable/explanation/pythonpath.html) it says
> It is highly recommended to arrange your test modules as packages by adding __init__.py files to your directories containing tests. This will make the tests part of a proper Python package, allowing pytest to resolve their full name (for example tests.core.test_core for test_core.py inside the tests.core package).

Adding a `tests/__init__.py` file made things work for me. Can you try the same? If we decide that that is the solution can you include a comment in the `__init__.py` file linking to the pytest docs above?

---

Yes. It is working. Thanks.